### PR TITLE
fix(popover): Fix size in iOS when rendered offscreen

### PR DESCRIPTION
**Pull request by pladaria**

### CODE REVIEW

File: src/popover.tsx [hunk 1]
```diff
@@ -116,8 +116,11 @@ const getWidthDesktop = (customWidth?: number): number => (customWidth ? customW
 const getPosition = (position: Position = defaultPositionDesktop, isTabletOrSmaller: boolean) =>
     isTabletOrSmaller && (position === 'left' || position === 'right') ? defaultPositionMobile : position;
 
-const getWidth = (isTabletOrSmaller: boolean, width?: number): number =>
-    isTabletOrSmaller ? window.innerWidth - marginLeftRightMobile * 2 : getWidthDesktop(width);
+const getWidth = (isTabletOrSmaller: boolean, isIos: boolean, width?: number): number =>
+    // in iOS, when the webview is rendered offscreen (eg. acccount tab), window.innerWidth value wrong, it returns strange values like 0 or 80.
+    isTabletOrSmaller
+        ? (isIos ? window.screen.width : window.innerWidth) - marginLeftRightMobile * 2
+        : getWidthDesktop(width);
```
Comment (pladaria) [inline src/popover.tsx:123]: Not very proud of this fix, but I was unable to find a better solution. 

I tried to listen to visibility changes and recalculate but doesn't work on the first render and the bug is reproduced

File: size-stats.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (atabel) [inline size-stats.json:1]: remove this file

**Review by atabel — CHANGES_REQUESTED** [review]

File: src/popover.tsx [hunk 3]
```diff
@@ -116,8 +116,11 @@ const getWidthDesktop = (customWidth?: number): number => (customWidth ? customW
 const getPosition = (position: Position = defaultPositionDesktop, isTabletOrSmaller: boolean) =>
     isTabletOrSmaller && (position === 'left' || position === 'right') ? defaultPositionMobile : position;
 
-const getWidth = (isTabletOrSmaller: boolean, width?: number): number =>
-    isTabletOrSmaller ? window.innerWidth - marginLeftRightMobile * 2 : getWidthDesktop(width);
+const getWidth = (isTabletOrSmaller: boolean, isIos: boolean, width?: number): number =>
+    // in iOS, when the webview is rendered offscreen (eg. acccount tab), window.innerWidth value wrong, it returns strange values like 0 or 80.
```
Comment (SergioMorchon) [inline src/popover.tsx:120]: lol

**Review by SergioMorchon — APPROVED** [review]

**Review by atabel — APPROVED** [review]

**Pull request merged**